hw14-warMvc/src/main/java/hw14/controller/AdminController.java

```diff
@@ -0,0 +1,24 @@
+package hw14.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
+import hw14.core.service.DBServiceUser;
+
+@Controller
+public class AdminController {
+    private final DBServiceUser dbServiceUser;
+
+    public AdminController(DBServiceUser dbServiceUser) {
+        this.dbServiceUser = dbServiceUser;
+    }
+
+    @GetMapping({ "/admin" })
+    public String userListView(Model model) {
```

Comment:
```suggestion
    @GetMapping({ "/users" })
    public String userListView(Model model) {
```

`userListView` должен быть в `UserController`.
`AdminController` лучше просто убрать. Он не нужен.

У вас сейчас как-то запутано получилось. Отображение списка в одном контроллере, добавление в другом и редирект.

Reply:
Убрал